tests/test_type.py

```diff
@@ -100,6 +100,22 @@ def test_check_airbnb():
     }
 
 
+def test_check_airports():
+    from vega_datasets import data
```

Comment:
`vega_datasets` is not a test dependency. We can just link to the dataset directly: 
```python
df = pd.read_csv('https://raw.githubusercontent.com/altair-viz/vega_datasets/master/vega_datasets/_data/airports.csv')
```

Reply:
Gotcha, thanks for the tip!